assets/css/src/components/_badge.scss

```diff
@@ -0,0 +1,10 @@
+.mailpoet-badge {
```

Comment:
Should all those sizes also come from variables? Why do we have the `grid` settings if we just keep using some random numbers in our CSS?

Reply:
I wouldn't do that. Not every single space is tied to the grid. We would end up creating grid variables for every pixel size just for the sake of using a variable and might end up with a much less readable code. In this case, `<Badge>` being a very specific visual component, I don't see a problem with it having its own spacing according to the design.

Reply:
Just to clarify, some of them could be replaced with already defined variables, but it wouldn't make sense to replace all. And if I should choose between `padding: 1px 8px 1px 20px;` and `padding: 1px $grid-gap-half 1px ($grid-gap-half + 12px);` I'd choose the first.

Reply:
Definitely not `($grid-gap-half + 12px)` that is no difference. I was thinking about adjusting this component so that it fits into the grid. 

> <Badge> being a very specific visual component

What is the rule then? 

I thought we are trying to do the redesign and we are using the grid for some reasons (consistency, visual cleanness, ...). If we keep having exceptions it undermines the purpose. The page will still look like a mess. But don't answer me I don't care that much about the design to argue about it. Just leave it as it is.

Reply:
The double border of the badge has a width of $grid-gap-medium thanks to that I was able to somehow define the spacing using grid variables.